examples/bitcoin_psbt/test/bitcoin.ts

```diff
@@ -0,0 +1,90 @@
+import { jsonParse } from 'azle';
```

Comment:
If not used removed, previous PR changes

Reply:
done